manifests/plugin/deb.pp

```diff
@@ -27,7 +32,14 @@
     $content = undef
   }
 
+  if $force_ignore_missing_package_indices {
+    $deb_plugin_config_fimpi = 'True'
+  } else {
+    $deb_plugin_config_fimpi = 'False'
+  }
```

Comment:
You can also write it as:
```suggestion
  $deb_plugin_config_fimpi = if $force_ignore_missing_package_indices { 'True' } else { 'False' }
```

Or, if you raise the lower bound of the stdlib dependency to 8.3.0 (in metadata.json):
```suggestion
  $deb_plugin_config_fimpi = to_python($force_ignore_missing_package_indices)
```